src/lib/components/docs/feedback-widget.svelte

```diff
@@ -104,25 +107,32 @@
               autocomplete="off"
               autocorrect="off"
               type="text"
-              class="mb-0"
+              class="mb-0 text-xs"
             />
-            <div>
-              <p class="text-sm my-4">
-                By submitting this form I acknowledge that I have read and
-                understood <a class="link" href="/privacy"
-                  >Gitpod’s Privacy Policy.</a
-                >
-              </p>
-              <span>
-                <Button
-                  variant="primary"
-                  size="medium"
-                  disabled={isSubmittedOnce}
-                  class="mt-micro"
-                  type="submit"><span>Send</span></Button
-                >
-              </span>
-            </div>
+            <Input
+              bind:value={email}
+              id="email"
+              name="email"
+              width="100%"
+              placeholder="Email(optional)"
+              class="text-xs"
+            />
+          </div>
+          <div class="mt-micro">
+            <p class="text-xs mb-x-small">
+              I consent to having this website store my submitted information so
+              that the support team can respond to my inquiry. For more
+              information, see our
+              <a class="link" href="/privacy"> Privacy Policy. </a>
```

Comment:
Since this is not our sales form. So, It might be good to use _`message`_ instead of _`inquiry`_

```suggestion
              I consent to having this website store my submitted information so
              that the support team can respond to my message. For more
              information, see our
              <a class="link" href="/privacy"> Privacy Policy. </a>
```